html/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Yoga as a Form of Medication</title>
+    </head>
+    <body>
+        <header>
+            <a name="top"></a>
+            <h1>Yoga as a Form of Medication</h1>
+        </header>
+        <nav>
+            <a href="#perfom">How to perform Yoga</a><br />
+            <a href="#meditation">Yoga Meditation</a><br />
+            <a href="#contact">Contact Us</a>
+        </nav>
+        <p>
+            Yoga is an ancient practice that originated in India over 5,000
+            years ago. It involves a combination of physical postures, breathing
+            exercises, meditation, and relaxation techniques. While yoga is
+            primarily known for its physical benefits, such as increased
+            flexibility and strength, it is also a powerful form of meditation.
+            Yoga as a form of medication involves using the practice of yoga to
+            improve physical and mental health. The physical postures, or
+            asanas, help to improve flexibility, strength, balance, and
+            circulation, while the breathing exercises, or pranayama, help to
+            calm the mind and improve respiratory function. The mental benefits
+            of yoga are well documented. Regular practice can help to reduce
+            stress, anxiety, and depression, and increase feelings of relaxation
+            and calm. This is because yoga works to balance the nervous system,
+            calming the sympathetic nervous system, which is responsible for the
+            fight or flight response, and activating the parasympathetic nervous
+            system, which is responsible for rest and relaxation. Yoga is an
+            ancient practice that combines physical, mental, and spiritual
+            elements to help individuals achieve a state of inner peace and
+            balance. This website explores the use of yoga as a form of
+            meditation and provides additional resources for those interested in
+            learning more.
+        </p>
+        <ul>
+            <li>
```

Comment:
From this line, you didn't nest your elements well. How can you put element li inside and outside. Please look at your code again